ui-tests/src/test/resources/features/integrations/api-provider.feature

```diff
@@ -238,6 +238,7 @@ Feature: API Provider Integration
       | task2 |
     Then wait until integration "TODO Integration get collection" gets into "Running" state
     When execute GET on API Provider route i-todo-integration-get-collection endpoint "/api/"
+    And wait until integration i-todo-integration-get-collection processed at least 1 message
```

Comment:
I don't think it's necessary to wait for this in the api provider tests - the above `execute GET` will trigger the integration flow, if it doesn't, there's no point in waiting for anything